evm/README.md

```diff
@@ -1,67 +1,172 @@
 
-## Usage
+_Events_
+``` solidity
+/// @notice Emitted whenn an outbound transfer is queued.
+/// @dev Topic0
+///      0x69add1952a6a6b9cb86f04d05f0cb605cbb469a50ae916139d34495a9991481f.
+/// @param queueSequence The location of the transfer in the queue.
+event OutboundTransferQueued(uint64 queueSequence);
+```
 
-### Build
+``` solidity
+/// @notice Emitted when an outbound transfer is rate limited.
+/// @dev Topic0
+///      0x754d657d1363ee47d967b415652b739bfe96d5729ccf2f26625dcdbc147db68b.
+/// @param sender The initial sender of the transfer.
+/// @param amount The amount to be transferred.
+/// @param currentCapacity The capacity left for transfers within the 24-hour window.
+event OutboundTransferRateLimited(
+    address indexed sender, uint64 sequence, uint256 amount, uint256 currentCapacity
+);
+```
 
-```shell
-$ cd evm && forge build
+``` solidity
+/// @notice Emitted when an inbound transfer is queued
+/// @dev Topic0
+///      0x7f63c9251d82a933210c2b6d0b0f116252c3c116788120e64e8e8215df6f3162.
+/// @param digest The digest of the message.
+event InboundTransferQueued(bytes32 digest);
 ```
 
-### Test
+3. **Transmit**
 
-```shell
-$ cd evm && forge test
+Once the NttManager forwards the message to the Transceiver, the message is transmitted via the [`sendMessage`] method. The method signature is enforced by the [`Transceiver`] but transceivers are free to determine their own implementation for transmitting messages. (e.g A message routed through the Wormhole Transceiver can be sent via automatic relaying (AR), via a specialized or custom relayer, or manually).
+
+Once the message has been transmitted, the contract emits the `SendTransceiverMessage` event.
+
+_Events_
+``` solidity
+/// @notice Emitted when a message is sent from the transceiver.
+/// @dev Topic0
+///      0x53b3e029c5ead7bffc739118953883859d30b1aaa086e0dca4d0a1c99cd9c3f5.
+/// @param recipientChain The chain ID of the recipient.
+/// @param message The message.
+event SendTransceiverMessage(
+    uint16 recipientChain, TransceiverStructs.TransceiverMessage message
+);
 ```
 
-### Format
+4. **Receive**
 
-```shell
-$ forge fmt
+Once a message has been transmitted across the wire, an off-chain process (e.g. a relayer) will forward the message to the corresponding Transceiver on the recipient chain. The relayer interacts with the transceiver via an entrypoint for receiving messages For example, the client will call the [`receiveWormholeMessage`] method on the WormholeTransceiver contract Wormhole to execute the message. Once the `ReceiveMessage` event is emitted during this process.
+
+This method should also forward the message to the NttManager on the recipient chain.
+Note that the the Transceiver interface does not enforce the method signature abstractly because receiving messages may be specific to the way in which a transceiver consumes messages.
```

Comment:
```suggestion
Note that the the Transceiver interface does not declare a signature for this method because receiving messages is specific to each transceiver, and a one size fits all solution would be overly restrictive.
```